data/energy_community_model.yml

```diff
@@ -163,14 +193,18 @@ users:  # list of users and the corresponding assets
   user4:  # name of the user
     # list of components
     
-    PV:  # component PV
-      type: renewable  # type of component
-      CAPEX_lin: 1400  # [€/kW] specific investment cost of the component
-      OEM_lin: 30  # [€/kW/y] specific O&M cost
-      lifetime_y: 25  # [y] lifetime of the component
-      max_capacity: 40  # [kW] maximum capacity
-      profile:  # list of profiles
-        ren_pu: pv  # code name of the specific renewable profile of the component
+    market_set: 
+      market_type: non_commercial #type of market
+      type: market  #type of market
+    
+    # PV:  # component PV
+    #   type: renewable  # type of component
+    #   CAPEX_lin: 1500  # [€/kW] specific investment cost of the component
+    #   OEM_lin: 30  # [€/kW/y] specific O&M cost
+    #   lifetime_y: 25  # [y] lifetime of the component
+    #   max_capacity: 3  # [kW] maximum capacity
+    #   profile:  # list of profiles
+    #     ren_pu: pv  # code name of the specific renewable profile of the component

```

Comment:
Re-enable the PV, here as in the following. The yaml shall be the same as before with only the addition of market_type

Reply:
Pushed now, sorry